### PR TITLE
Documentation

### PR DESCRIPTION
**Pull request by mansipadgelwar**
Add documentation page for component library.

### CODE REVIEW

**Review by SadiqZak — APPROVED** [review]
Yup, looks good to merge!

File: documentation/documentation.css [hunk 1]
```diff
@@ -0,0 +1,149 @@
...
+
+.side_bar{    
+    width: 16rem; 
+    position: sticky;
+    overflow-y: auto;
+    top: 0;
+    background-color: white;
+    text-align: center;
+    height: calc(100vh - 2rem);    
+}
```
Comment (thegeorgenikhil) [inline documentation/documentation.css:85]: A bit of sidebar is visible on mobile screens, better to use a media query for smaller screens and make the  `.side_bar` to `display:none`

**Review by thegeorgenikhil — APPROVED** [review]

File: documentation/documentation.css [hunk 2]
```diff
@@ -0,0 +1,149 @@
...
+
+.list-item-inline {
+    display: inline;
+}
+
+.documentation-container{
+    display: flex;
+    flex-direction: column;
+    gap: 20px;
```
Comment (tejas-muthya37) [inline documentation/documentation.css]: 2.25 rem here instead of 20px might help you maintain consistency in units.

**Review by srijani07 — APPROVED** [review]

**Review by tejas-muthya37 — APPROVED** [review]
Rest of the code looks good to me. Approving this PR. Good luck :)

**Pull request merged**